src/Network/Minio/Data.hs

```diff
@@ -27,7 +27,7 @@ data ConnectInfo = ConnectInfo {
   } deriving (Eq, Show)
 
 instance Default ConnectInfo where
-  def = ConnectInfo "localhost" 9000 "minio" "minio123" False "us-east-1"
+  def = ConnectInfo "play.minio.io" 9000 "Q3AM3UQ867SPQQA43P2F" "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG" True "us-east-1"
```

Comment:
After thinking about this, I think we can go with Minio Play as the default server for tests - it works more simply though it is a bit slow. However, we can add a flag to use a local server for development (I'll take care of this part).